bitnami/ghost/README.md

```diff
@@ -139,6 +139,7 @@ The following table lists the configurable parameters of the Ghost chart and the
 | `podSecurityContext.fsGroup`            | Ghost pods' group ID                                                                                                  | `1001`                                      |
 | `priorityClassName`                     | Define the priority class name to use for the ghost pods here.                                                        | `""`                                        |
 | `replicaCount`                          | Number of Ghost Pods to run                                                                                           | `1`                                         |
+| `strategy.type`                         | Kubernetes deployment strategy type                                                                                   | `RollingUpdate`                             |
```

Comment:
Could you please check the template that we have in order to follow the conventions for these kind of parameters? If you check it, the parameter should be named `updateStrategy`. Here you have how this parameter is expected to be used:

https://github.com/bitnami/charts/blob/995271aa871117ae2cceb8ee30aa4eda40380101/template/CHART_NAME/values.yaml#L216-L222

https://github.com/bitnami/charts/blob/995271aa871117ae2cceb8ee30aa4eda40380101/template/CHART_NAME/templates/deployment.yaml#L15-L17

Reply:
Thank you for the review!
I've applied requested changes.
A few notes:
 - There is already `updateStrategy` parameter in `Readme.md` and looks like it was deleted but left in the file for some reason.
 - The line from deployment template `strategy: {{- toYaml .Values.co%%MAIN_OBJECT_BLOCK%%ector.updateStrategy | nindent 4 }}` seems broken because of `co%%MAIN_OBJECT_BLOCK%%ector`. Is this expected to be  `strategy: {{- toYaml .Values.%%MAIN_OBJECT_BLOCK%%updateStrategy | nindent 4 }}`?

Reply:
Yes, it is a typo, thanks for letting us know. I just fixed it in this commit https://github.com/bitnami/charts/commit/03db52dcc4e858d1f58bc2fc4ca51674ad2d0011